src/coreclr/inc/corinfo.h

```diff
@@ -2325,18 +2325,9 @@ class ICorStaticInfo
             CORINFO_CLASS_HANDLE    cls
             ) = 0;
 
-    virtual CORINFO_MODULE_HANDLE getClassModule (
-            CORINFO_CLASS_HANDLE    cls
-            ) = 0;
-
-    // Returns the assembly that contains the module "mod".
-    virtual CORINFO_ASSEMBLY_HANDLE getModuleAssembly (
```

Comment:
This was the only use of `CORINFO_ASSEMBLY_HANDLE` on JIT/EE interface. Can be deleted now?